components/builder-web/app/origin/origin-page/origin-members-tab/origin-members-tab.component.ts

```diff
@@ -89,6 +97,10 @@ export class OriginMembersTabComponent implements OnInit, OnDestroy {
     return this.store.getState().session.token;
   }
 
+  private get isPrivate() {
+    return this.store.getState().origins.current.default_package_visibility === 'private' ? true : false;
+  }
```

Comment:
explicitly used in the callback after a user departs to determine where they should be routed.